src/clr-angular/data/datagrid/render/cell-renderer.ts

```diff
@@ -40,12 +45,19 @@ export class DatagridCellRenderer implements OnDestroy {
   }
 
   private stateChanges(state: DatagridColumnState) {
+    if (this.runAllChanges) {
```

Comment:
No test case for running all changes, can we add something?

Reply:
Done